play-services-core/src/main/kotlin/org/microg/gms/auth/credentials/assistedsignin/AssistedSignInService.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-FileCopyrightText: 2023 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.microg.gms.auth.credentials.assistedsignin
```

Comment:
Please use package `org.microg.gms.auth.credentials.identity` and name `IdentitySignInService` to match AIDL, public API and existing naming schemes.

Reply:
applied